src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1689,4 +1689,13 @@
   <data name="StaticAndAbstractRequiresPreviewFeatures" xml:space="preserve">
     <value>Using both 'static' and 'abstract' modifiers requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
   </data>
+  <data name="ModuleInitializerAttributeShouldNotBeUsedInLibrariesDescription" xml:space="preserve">
+    <value>Module initializers are intended to be used by application code to ensure an application's components are initialized before the application code begins executing. If library code declares a [ModuleInitializer] method, it can interfere with application initialization and also lead to limitations in that application's trimming abilities. Library code should therefore not utilize the [ModuleInitializer] attribute, but instead expose methods that can be used to initialize any components within the library and allow the application to invoke the method during application initialization.</value>
+  </data>
+  <data name="ModuleInitializerAttributeShouldNotBeUsedInLibrariesMessage" xml:space="preserve">
+    <value>The ModuleInitializer attribute is only intended to be used in application code or advanced source generator scenarios</value>
```

Comment:
Usually we enclose type names between single quotes (applies to `Message`, `Title`, and `Description`)